templates/less/pages/reset-password.less

```diff
@@ -0,0 +1,56 @@
+.requestPassword {
+  height: 180px;
+  padding-top: 26px;
+  .btn.btn-awsm {
+    max-width: 185px;
+  }
+}
+
+.resetPassword {
```

Comment:
the entire rest of this file should be *inside* the page's namespacing block. There should be nothing outside